zebra-chain/src/orchard/address.rs

```diff
@@ -37,13 +33,8 @@ impl Arbitrary for Address {
     type Parameters = ();
```

Comment:
`impl Arbitrary for Address` needs to also be active if the `proptest-impl` feature is enabled.

Let's make this change for all the files modified in this PR.